core/domain/skill_domain_test.py

```diff
@@ -83,10 +83,20 @@ def _assert_valid_skill_id(self, expected_error_substring, skill_id):
             utils.ValidationError, expected_error_substring):
             skill_domain.Skill.require_valid_skill_id(skill_id)
 
+    def _assert_valid_skill_ids(self, expected_error_substring, skill_ids):
+        """Checks that the skill passes strict validation."""
+        with self.assertRaisesRegexp(
+            utils.ValidationError, expected_error_substring):
+            skill_domain.Skill.require_valid_skill_ids(skill_ids)
+
     def test_valid_skill_id(self):
         self._assert_valid_skill_id('Skill id should be a string', 10)
         self._assert_valid_skill_id('Invalid skill id', 'abc')
 
+    def test_valid_skill_ids(self):
```

Comment:
Use better test names (as discussed before). You can update the other ones in this class/file too, if you like -- it would be nice to do, but it is optional.

Reply:
Fixed the upper one also(since my test was inspired by the upper one). 
Done.